src/Components/Message/Modal/FullscreenModal.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<Modal :show="show" :show-close-button="showCloseButton" @close="close">
+	<Modal ref="fullscreen" class="c-modal-fullscreen" :show="show" :show-close-button="showCloseButton" @close="close">
```

Comment:
```suggestion
	<Modal ref="fullscreen" class="c-fullscreen-modal" :show="show" :show-close-button="showCloseButton" @close="close">
```
FullscreenModal 컴포넌트에 대한 루트 클래스니까 이렇게 바꾸는게 맞겠네요